### PR TITLE
UG updates, ResultDisplay text wrapping

### PR DESCRIPTION
**Pull request by daytona65**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -29,9 +37,11 @@ on every page to take you back to the Table of Contents. To further augment your
 
 1. Copy the file to the folder you want to use as the _home folder_ for your MedInfo.
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar medinfo.jar` command to run the application.<br>
-   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+1. Run the application by
+   2. Double-clicking the jar file or 
+   2. Opening a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar medinfo.jar` command to run the application.<br>
+1. A GUI similar to the one below should appear in a few seconds. Note how the app contains some sample data.<br>
```
Comment (nramapurath) [inline docs/UserGuide.md:43]: Is the list numbering here incorrect or is it bad formatting on my end?

File: docs/_sass/minima/_base.scss [hunk 2]
```diff
@@ -288,8 +289,11 @@ table {
     text-align: center;
   }
   .site-header:before {
-    content: "AB-3";
+    content: "MedInfo";
```
Comment (nramapurath) [inline docs/_sass/minima/_base.scss:292]: Nice find on fixing the UG title!

**Review by nramapurath — CHANGES_REQUESTED** [review]
Good changes! Fixes our project scope and hope to see it implemented... Just a couple nits to fix

File: docs/UserGuide.md (hunk 1 above)
Comment (daytona65) [reply docs/UserGuide.md:43]: if you see the numbering on my site its fine:

**Pull request merged**